### DISCUSSION TITLE
pdf export configuration

### DISCUSSION THREAD

**Discussion by bnlawrence** [chrisgrieser/shimmering-focus#222] [Q&A] [answered]
I really like the export pdf that is exposed in shimmering focus, and the configuration options that exist.

I have two questions though.

1. Internal links . Is it possible to turn off the "internal links" and keep only external links? I see that pandoc export has an option to replace internal links with text, and it would be good if that were possible here too ... 

2. Embedded notes. At the moment, by default embedded notes appear embedded. It'd be good if for export, when we embed a sequence of notes, the document looks contiguous ,that is, in the exported PDF it was not obvious that it was constructed from multiple notes.  Is that possible, or could it be made possible? I suspect the solution to his is in the CSS, and would be happy to (eventually) look at offering it as an option via snippet if someone could point me to the right place in theme-land.

**Comment by chrisgrieser** [comment, marked as answer]
> I really like the export pdf that is exposed in shimmering focus, and the configuration options that exist.

tbh, I haven't styled anything about the PDF export 😅 
Since I do not use the PDF export at all, I unfortunately do not have any plans to do styling for PDF export, sorry!

The styling that does exist is most likely only a side effect of the existing styling and/or the styling from the default theme.

**Reply by bnlawrence** [reply]
Ok ta, will see if I can work out where and how it's done (eventually 😀)
